.gitignore

```diff
@@ -1 +1,4 @@
-node_modules
\ No newline at end of file
+node_modules
+yarn-error.log
+lerna-debug.log
+yarn.lock
```

Comment:
should we be ignoring  the yarn.lock file? this is the only thing that is stoping me for approving this PR cc @gh0stl1m

Reply:
In my opinion, we should keep this file in the repo.

For a better explanation, visit the next link:
https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for#answer-40206145